### PR TITLE
Adding links to footer on paywall

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding links to footer on paywall.
I also took that chance to refactor and DRY up things...

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-protocol.com/src/static/blog.rss [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <description><![CDATA[News and updates from the Web's new business model.]]></description>
         <link>https://unlock-protocol.com/blog</link>
         <generator>Unlock Blog Engine</generator>
-        <lastBuildDate>Tue, 17 Dec 2019 23:16:19 GMT</lastBuildDate>
+        <lastBuildDate>Wed, 18 Dec 2019 01:27:42 GMT</lastBuildDate>
```
Comment (cnasc) [inline unlock-protocol.com/src/static/blog.rss]: Should this be in here?

**Comment by cnasc** [comment]
And I believe this will fix #5505 ?

File: unlock-protocol.com/src/static/blog.rss (hunk 1 above)
Comment (julien51) [reply unlock-protocol.com/src/static/blog.rss]: Niope.

**Comment by julien51** [comment]
and #5454

**Pull request merged**